.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["2.7", "3.11"]
```

Comment:
GitHub Actions no longer supports Python 2, so this must wait to merge in until after a 2->3 migration is done.